Tests/Functional/Asynchronous/Predicates/Expectations/main.swift

```diff
@@ -0,0 +1,83 @@
+// RUN: %{swiftc} %s -o %{built_tests_dir}/Asynchronous-Predicates
+// RUN: %{built_tests_dir}/Asynchronous-Predicates > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(Linux) || os(FreeBSD)
+    import XCTest
+    import Foundation
+#else
+    import SwiftXCTest
+    import SwiftFoundation
+#endif
+
+// CHECK: Test Suite 'All tests' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Suite '.*\.xctest' started at \d+:\d+:\d+\.\d+
+
+// CHECK: Test Suite 'PredicateExpectationsTestCase' started at \d+:\d+:\d+\.\d+
+class PredicateExpectationsTestCase: XCTestCase {
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyTruePredicateAndObject_passes' started at \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'PredicateExpectationsTestCase.test_immediatelyTruePredicateAndObject_passes' passed \(\d+\.\d+ seconds\).
```

Comment:
With #94 merged you may now indent the `// CHECK:`, like so:

```swift
func myFunc() {
    // CHECK: foo
    print("foo")
    if (bar) {
        // CHECK: baz
        print("baz")
    }
}
```

I think indenting is more readable, so I'd encourage you to do that! (I had been meaning to start sending out pull requests to indent the checks in the tests, but forgot 😇 )